mkdocs.yml

```diff
@@ -21,11 +21,16 @@ nav:
       - Glossary: 'overview/glossary.md'
       - Logo assets: 'overview/logos.md'
   - Getting Started:
-      - Getting Started: 'getting-started/index.md'
-      - Create an App: 'getting-started/create-an-app.md'
-      - App configuration:
+      - Installing a standalone server (Developer or Admin): 'getting-started/standalone-install.md'
+      - Configuring Backstage (Admin):
+          - Database: 'getting-started/config/database.md'
+          - Authentication: 'getting-started/config/authentication.md'
           - Configuring App with plugins: 'getting-started/configure-app-with-plugins.md'
           - Customize the look-and-feel of your App: 'getting-started/app-custom-theme.md'
+          - Customizing your Homepage: 'getting-started/homepage.md'
+      - Logging in: 'getting-started/logging-in.md'
+      - Register a component: 'getting-started/register-a-component.md'
+      - Create a component: 'getting-started/create-a-component.md'
       - Keeping Backstage Updated: 'getting-started/keeping-backstage-updated.md'
       - Key Concepts: 'getting-started/concepts.md'
       - Project Structure: 'getting-started/project-structure.md'
```

Comment:
These are also arguably not getting started and should be moved elsewhere.

Reply:
Also the same for `Contributors` too, can we move them :pray:

Reply:
Moved to a new `Contribute` section, I was trying to keep this PR focused but it makes sense to shift 'em. Currently, the new section is waaaay down on the sidebar, but with the other TOC updates to come, it should surface as an easy to find element.